python/dgl/sparse/sparse_matrix.py

```diff
@@ -586,6 +586,93 @@ def range_select(self, dim: int, index: slice):
             )
         raise TypeError(f"{type(index).__name__} is unsupported input type.")
 
+    def sample(
+        self,
+        dim: int,
+        n_pick: int,
+        ids: Optional[torch.Tensor] = None,
```

Comment:
```suggestion
        nodes: Optional[torch.Tensor] = None,
```

Reply:
I thinks `ids` is more suitable for SparseMatrix, which does not have the concepts of `graph` or `node`.

Reply:
SGTM.